chain/consensus/filcns/upgrades.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/onsi/ginkgo/reporters/stenographer/support/go-isatty"
```

Comment:
Instead of a new dependency, we could just use `"golang.org/x/term"`:

```go
term.IsTerminal(int(os.Stderr.Fd()))
```

Which is already used in cli/wallet.go

Or, `"github.com/mattn/go-isatty"`

which is already used in cli/util.go, cmd/lotus-miner/info.go, cmd/lotus/main.go and a bunch of dependencies we pull in, like go-log:

```go
isatty.IsTerminal(os.Stderr.Fd())
```

Reply:
thanks, updated